lib/ButtonLink.js

```diff
@@ -1,6 +1,6 @@
-var React = require('react');
+var React = require('react/addons');
 
-var Button = require('react-bootstrap/lib/Button');
+var Button = require('react-bootstrap').Button;
```

Comment:
By bringing in `require('react-bootstrap')` this will invariably bring in the entire react-bootstrap library, when all you need is the button. In this case tying this to the 0.15.1 release would be desired.